charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -44,6 +44,8 @@ Arguments (dict):
   value: parentbased_traceidratio
 - name: OTEL_TRACES_SAMPLER_ARG
   value: "0.1"
+- name: OTEL_EXPORTER_OTLP_ENDPOINT
+  value: {{.root.Values.collector | quote}}
```

Comment:
Can we put that somewhere into the tracing object? Instead of making it a root level value?

How do we deal with a missing value?

Reply:
> Can we put that somewhere into the tracing object? Instead of making it a root level value?

I'll try that :+1: 

> How do we deal with a missing value?

I think we can add 1 `if`